utility/src/main/java/org/oppia/android/util/profile/BUILD.bazel

```
@@ -0,0 +1,16 @@
+"""
+General purposes utilities to manage Directories.
```

```suggestion
General purposes utilities to manage directories.
```

No need to capitalize here.

---

@BenHenning Ok I removed the capitalization.